charts/nginx-gateway-fabric/values.schema.json

```diff
@@ -90,9 +90,11 @@
                   "items": {
                     "properties": {
                       "key": {
+                        "pattern": "^([^\"$\\\\]|\\\\[^$])*$",
```

Comment:
Does this behave right? That's a lot of escaping.

Reply:
tested them and seem to be working fine